src/main/resources/checkstyle.xml

```diff
@@ -95,12 +95,30 @@
         <property name="message" value="Use JUnit v4 instead."/>
     </module>
 
-    <!-- Forbid shaded packages -->
+    <!-- Always throw propagate result -->
     <module name="RegexpSingleline">
         <property name="format" value="^[ ]*Throwables.propagate\(.*"/>
```

Comment:
We can make this change separately. This PR is to keep the private repo in-sync with the public one.